CHANGES.rst

```diff
@@ -12,6 +12,9 @@
   that argument. See `issue 208
   <https://github.com/zopefoundation/zope.interface/issues/208>`_.
 
+- Make sure to call each invariant only once when validating invariants. 
```

Comment:
I think the indentation is probably off here, causing [the failures of `twine check`](https://travis-ci.org/github/zopefoundation/zope.interface/jobs/730943537#L339). Sorry about that.